exports/security-group-variables.tf

```diff
@@ -1,6 +1,6 @@
-# security-group-inputs Version: 2
+# security-group-variables Version: 2
```

Comment:
should this version be bumped and use semver ?

Reply:
prob not related to the file name, but to the file content, which did not change